src/createFabricMap.tsx

```diff
@@ -41,213 +41,217 @@ export interface FabricMapHandle {
 }
 
 const createFabricMap = (ViewComponent: React.ComponentType, Commands: any) => {
-  return forwardRef<FabricMapHandle, FabricMapViewProps>((props, ref) => {
-    const fabricRef = useRef<React.ElementRef<typeof ViewComponent>>(null);
+  return forwardRef<FabricMapHandle | null, FabricMapViewProps>(
```

Comment:
Scary diff head! But this `| null` bit is actually the only change, the rest is Prettier's work.